doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+* has a different *status and style* of the project. EEGLAB is the biggest EEG analysis package. MNE is a comparatively young project. It follows modern coding standards (including code review and unit tests). Its development is less monolithic, more distributed and democratic than that of EEGLAB. There are many published papers using EEGLAB, and very few with MNE-Python, and many users of EEGLAB and few of MNE.
+
+Minor differences: MNE-Python ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* is much more developed with regards to **MVPA/decoding analyses**. It is also developed with a close eye on the leading Machine Learning toolbox *scikit-learn*.
+* 's *ICA is not as deeply integrated* into the system. Support for clustering ICs is very rudimentary. However, MNE's ICA implementations are fully featured, and in some ways superior to EEGLAB. For example, MNE's infomax implementation can be much faster than EEGLAB's, in large part due to multicore usage. Also, MNE ICA objects allow easily applying the same ICA fit to multiple data sets (e.g., it is easy to fit ICA on high pass filtered data and apply it to unfiltered data).
+* does not feature *plugins* as strongly as EEGLAB does. No true equivalents exist for high quality plugins such as LIMO or ERPLAB. Some features which are plugins in EEGLAB are parts of base MNE, such as CORRMAP, and many plugins are at least partially mirrored by MNE core functionality (although ironically, not Alex Gramfort's graph cut plugin for single trial analysis!!!).
+* does not have as broad *data system format support* for EEG data formats as EEGLAB does with `fileio`. However, MNE can read many common file formats, including the EEGLAB format, and of course many MEG formats.
+* does not have the extensive inbuilt *single-trial* capabilities of EEGLAB. For example, while it is possible to plot single trials sorted by alpha power by feeding mne.viz.plot_epochs_image a function built around psd_epochs, it requires much more user input.
+* does not easily allow the rich storage of epochs-level *event information* EEGLAB has. MNE events are stored in the three-column `events` format, all integers.
+* does not have an equivalent to EEGLAB's *STUDY* structure. Grand averages of single-subject `evoked` and `average_tfr` objects can be constructed with the `mne.grand_average` function (beware of the default setting of weighting by trials), which return regular `evoked` and `average_tfr` objects. Individual subjects can simply be stored in lists or dicts of `epochs` or `raw` objects.
+
+Partial EEGLAB/MNE translation table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+============================================================================================================================
```

Comment:
you don't need pep8 for these. It's only for python code. However sometimes, people prefer to break the lines so that git diffs look more clean.

This will be of help: http://sphinx-doc.org/rest.html